Core/Utilities/RabbitMQAdmin.py

```diff
@@ -0,0 +1,138 @@
+"""RabbitMQAdmin module serves for the management of the internal RabbitMQ
+   users database. It uses rabbitmqctl command. Only the user with the right
+   permissions can execute those commands.
+"""
+import re
+from DIRAC import S_OK, DError
+import errno
+from DIRAC.Core.Utilities import Subprocess
+
+def executeRabbitmqctl(arg, *argv):
+  """Executes RabbitMQ administration command.
+    It uses rabbitmqctl command line interface.
+    For every command the -q argument ("quit mode")
+    is used, since in some cases the output must be processed,
+    so we don't want any additional informations printed.
+  Args:
+    arg(str): command recognized by the rabbitmqctl.
+    argv(list): optional list of string parameters.
+  Returns:
+    S_OK:
+    Derror:
+
+  """
+  command =['sudo','/usr/sbin/rabbitmqctl','-q', arg] + list(argv)
```

Comment:
Because it is explicit :-) It is more psychologic that programmatic. If people have to conciously write "sudo", they reallize they are performing an admin operation which might have heavy consequences. Embarquing it in script is dangerous from this point of view, especially with the "NOPASSWD" policy of the sudoers file